nmstatectl/nmstatectl.py

```diff
@@ -20,6 +20,7 @@
 from __future__ import absolute_import
```

Comment:
Commit message:
Can you please use "filter" instead of "limit"?
It will be also useful if you give an example in the commit message on options to use it.

Reply:
done